lib/dialects/mariadb/query-generator.js

```diff
@@ -24,8 +24,12 @@ class MariaDBQueryGenerator extends MySQLQueryGenerator {
     return `SELECT SCHEMA_NAME as schema_name FROM INFORMATION_SCHEMA.SCHEMATA WHERE SCHEMA_NAME NOT IN ('MYSQL', 'INFORMATION_SCHEMA', 'PERFORMANCE_SCHEMA'${skip ? skip.reduce( (sql, schemaName) => sql +=  `,${this.escape(schemaName)}`, '') : ''});`;
   }
 
-  showTablesQuery() {
-    return 'SELECT TABLE_NAME, TABLE_SCHEMA FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_SCHEMA NOT IN (\'MYSQL\', \'INFORMATION_SCHEMA\', \'PERFORMANCE_SCHEMA\') AND TABLE_TYPE = \'BASE TABLE\'';
+  showTablesQuery(database) {
+    let query = 'SELECT TABLE_NAME, TABLE_SCHEMA FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_SCHEMA NOT IN (\'MYSQL\', \'INFORMATION_SCHEMA\', \'PERFORMANCE_SCHEMA\') AND TABLE_TYPE = \'BASE TABLE\'';
```

Comment:
I would like to get feedback from @rusher since he created the PR for MariaDB (#10192).

I have a couple questions about your initial PR:

1. Was it intentional to return all tables in all databases?
2. If MariaDB has schemas separate from databases why is `TABLE_SCHEMA` the same as the database in `INFORMATION_SCHEMA.TABLES`?

Reply:
https://dev.mysql.com/doc/refman/8.0/en/create-database.html
`CREATE SCHEMA is a synonym for CREATE DATABASE.`

Reply:
It seems to be the same for MariaDB

https://mariadb.com/kb/en/library/create-database/

I've never used MariaDB and I'm starting to wonder if it actually supports schemas the same way PostgreSQL and MSSQL do.